week2/countword/file/file.go

```diff
@@ -0,0 +1,57 @@
+package file
```

Comment:
I suggest you should name this package is `fileutil` or `util` because we may use `file` as a variable, this will cause some misunderstanding.